trojsten/special/plugin_prask_5_1_1/core.py

```diff
@@ -70,20 +192,14 @@ def run(cls, x, try_count):
 
 class Level6(object):
 
-    TARGET = "Oto"
-    DATA = []
-    FILE = os.path.join(os.path.dirname(__file__), "mena.txt")
-
-    @classmethod
-    def init_data(cls):
-        if not cls.DATA:
-            with codecs.open(cls.FILE, "r", "utf8") as f:
-                cls.DATA = list([name.strip() for name in f])
+    TARGET = "3684591"
 
     @classmethod
     def run(cls, x, try_count):
-        cls.init_data()
-        return cls.DATA[x % 365]
+        ans = ''
+        for (i, c) in enumerate(str(x)):
+            ans += str((((int(c) - i) % 10) + 10) % 10)
```

Comment:
Tomuto daj funkciu, nemám ani šajnu čo má toto znamenať. Also, čo je `i`, `c` a `cls`?

Reply:
Nedam tomu funkciu lebo krajsie to zapisat nejde, je to i-ta cifra - i modulo 10

Reply:
Tak urob funkciu `i_ta_cifra`.